lib/datasource/index.ts

```diff
@@ -372,23 +373,26 @@ export async function getPkgReleases(
   return res;
 }
 
-export function supportsDigests(config: DigestConfig): boolean {
-  return 'getDigest' in getDatasourceFor(config.datasource);
+export function supportsDigests(datasource: string | undefined): boolean {
```

Comment:
Why is this change required?

Reply:
Caller of this function passes the type which is now incompatible with `DigestConfig`